### PR TITLE
Add constants

### PR DESCRIPTION
**Pull request by JoshuaSBrown**
Adding explicit constants for dealing with molecule, residue, element type and ids when they have not been assigned.

### CODE REVIEW

**Comment by junghans** [comment]
Maybe we can use these new variables in a unittest...

**Comment by JoshuaSBrown** [comment]
Sure is this what code cov does? It forces you to have decent coverage of new code that you add?

**Comment by junghans** [comment]
@JoshuaSBrown codecov isn't required, but it gives an indicator if added code has is covered by the tests.

File: src/tests/CMakeLists.txt [hunk 1]
```diff
@@ -2,7 +2,8 @@ find_package(Boost 1.53.0 REQUIRED COMPONENTS unit_test_framework)
 
 # Each test listed in Alphabetical order
 foreach(PROG
-    test_cubicspline
+		test_constants
+		test_cubicspline
```
Comment (junghans) [inline src/tests/CMakeLists.txt]: Indentation?

**Comment by junghans** [comment]
There is no doxygen documentation of the new members.

**Comment by JoshuaSBrown** [comment]
I'm not quite sure what needs to be added, the names of the variables cann't exactly get much more explicit.

**Comment by junghans** [comment]
Well, you did it now - I will the idea of slowly increasing the doxygen coverage over time.

**Pull request merged**